tracer/src/Datadog.Trace/Debugger/Expressions/ProbeExpressionEvaluator.cs

```diff
@@ -190,6 +206,85 @@ private void EvaluateMetric(ref ExpressionEvaluationResult result, MethodScopeMe
         }
     }
 
+    private void EvaluateSpanDecorations(ref ExpressionEvaluationResult result, MethodScopeMembers scopeMembers)
+    {
+        var resultBuilder = new List<ExpressionEvaluationResult.DecorationResult>();
+        try
+        {
+            EnsureNotNull(_compiledDecorations);
+
+            var compiledDecorations = _compiledDecorations.Value;
+            if (compiledDecorations == null)
+            {
+                Log.Debug($"{nameof(ProbeExpressionEvaluator)}.{nameof(EvaluateSpanDecorations)}: {nameof(_compiledDecorations.Value)} is null");
```

Comment:
The way you're using the logger is not on par as we use it throughout the code base.
Please review Andrew's PRs:
1. [#3785 / Our debugger files](https://github.com/DataDog/dd-trace-dotnet/pull/3785/files#diff-dfc41c32a4ead9325d8042f807b9b788006bb4b09d29a4005988badeef30edf4).
2. [#4140 / Our debugger files](https://github.com/DataDog/dd-trace-dotnet/pull/4140/files#diff-43b530bbc16d91d2ffee0ae992cf23dc30b5edfe70dc7e4896f0ecd90e013ef6).

You should not be using string interpolation, but instead rely on the formatting of Serilog.

I would also suggest surrounding it with a check if Debug log level is enabled to spare the string allocation.